src/translations/lang/en.json

```diff
@@ -1041,6 +1041,8 @@
     "TX_QUERY_DEFAULT_TOOLTIP": "You've arrived here from a pre-filled transaction link. Make sure the settings are correct before continuing. [Learn more](https://support.mycrypto.com/how-to/sending/creating-prefilled-transaction).",
     "TX_QUERY_SPEED_UP_TOOLTIP": "This transaction will speed up a previously pending transaction with the same nonce. It does so by sending the same transaction but with a higher gas price.",
     "TX_QUERY_CANCEL_TOOLTIP": "This transaction aims to cancel a previously pending transaction. It does so by sending a 0.00 ETH transaction to yourself (with the same nonce as the previous transaction) with the purpose of being mined before the original transaction.",
-    "SENDING_TO_TOKEN_ADDRESS": "You are sending to the $token contract. If you want to send $token to an address, change the Recipient to the address you want to send the $token to. If you send value to the $token contract itself, it will likely be irrecoverable."
+    "SENDING_TO_TOKEN_ADDRESS": "You are sending to the $token contract. If you want to send $token to an address, change the Recipient to the address you want to send the $token to. If you send value to the $token contract itself, it will likely be irrecoverable.",
+    "LEDGER_NANO_S_NEW_HELP": "Not working? [Here's some troubleshooting tips to try](https://support.mycrypto.com/troubleshooting/accessing-wallet/ledger-hardware-wallet-unable-to-connect-on-mycrypto)",
+    "LEDGER_NANO_S_NEW_REFERRAL": "Need a Ledger? [Get one now.](https://www.ledgerwallet.com/r/1985?path=/products/)"
```

Comment:
Huh?

Reply:
I noticed they weren't translating in the ledger sign tx component since the translation functions composes the translation key from the wallet id [here](https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/components/SignTransactionWallets/Hardware.tsx#L115-L122).